src/providers/completion/alloyAutoCompleteRules.ts

```diff
@@ -77,12 +77,12 @@
 	requireRange: true,
 	rangeRegex: /([\w/.$]+)/,
 	async getCompletions (projectDir, range) {
-		const rootPath = await projectDir.isAlloyProject() ? path.join(projectDir.filePath) : projectDir.filePath;
+		const rootPath = await projectDir.isAlloyProject() ? path.join(projectDir.filePath, 'app') : projectDir.filePath;
 		const assetPath = path.join(rootPath, 'assets');
 		const completions: CompletionItem[] = [];
 		// limit search to these sub-directories
-		let paths = [ 'images', 'iphone', 'android', 'windows' ];
-		paths = paths.map(aPath => path.join(assetPath, aPath));
+		let paths = [ 'images', 'iphone', 'android' ];
+		paths = [ ...paths.map(aPath => path.join(assetPath, aPath)), assetPath];
```

Comment:
<!--
  0 failure: 
  1 warning:  src/providers/com...
  
  
  DangerID: danger-id-Danger;
  File: src/providers/completion/alloyAutoCompleteRules.ts;
  Line: 85;
-->

- :warning: src/providers/completion/alloyAutoCompleteRules.ts line 85 – A space is required before ']'. (array-bracket-spacing)